subsys/bootloader/bl_monotonic_counters/bl_monotonic_counters.c

```diff
@@ -122,22 +111,28 @@ static uint16_t get_counter(const uint16_t **free_slot)
 	if (free_slot != NULL) {
 		*free_slot = addr;
 	}
-	return highest_counter;
-}
 
+	*counter_value = highest_counter;
+	return 0;
```

Comment:
Doc says we are returning the current value of the counter, but here we return 0.

Reply:
The doc will be updated